cl_sii/rtc/data_models.py

```diff
@@ -47,6 +47,23 @@ def validate_cesion_seq(value: int) -> None:
         raise ValueError("Value is out of the valid range.", value)
 
 
+def validate_cesion_fecha(value: datetime, tz: tz_utils.PytzTimezone) -> None:
+    """
+    Validate value of date and time when the "cesión" happened.
+
+    :raises ValueError:
+    """
+
+    tz_utils.validate_dt_tz(value, tz)
+
+    now_tz_aware = tz_utils.get_now_tz_aware()
+
+    if not (value.date() <= now_tz_aware.date()):
```

Comment:
I think this is not correct because the output of `now_tz_aware.date()` depends on the TZ of `now_tz_aware`, which has not been set.

Reply:
I agree. Let's bring back `current_date_in_tz = tz_utils.get_now_tz_aware().astimezone(tz)`.